client/trino-cli/src/main/java/io/trino/cli/QueryRunner.java

```diff
@@ -20,6 +20,7 @@
 import io.trino.client.StatementClient;
 import io.trino.client.auth.external.ExternalAuthenticator;
```

Comment:
In commit message: `externalAuthenticationTokenCache=CASHED` did you mean `CACHED`? I think we should use `externalAuthenticationTokenCache=MEMORY`